spec.html

```diff
@@ -9562,31 +9551,31 @@ <h1>Module Namespace Exotic Objects</h1>
 
       <emu-clause id="sec-module-namespace-exotic-objects-setprototypeof-v">
         <h1>[[SetPrototypeOf]] ( _V_ )</h1>
-        <p>When the [[SetPrototypeOf]] internal method of a module namespace exotic object _O_ is called with argument _V_, the following steps are taken:</p>
+        <p>The [[SetPrototypeOf]] internal method of a module namespace exotic object _O_ takes argument _V_ (an Object or *null*). It performs the following steps when called:</p>
         <emu-alg>
           1. Return ? SetImmutablePrototype(_O_, _V_).
         </emu-alg>
       </emu-clause>
 
       <emu-clause id="sec-module-namespace-exotic-objects-isextensible">
         <h1>[[IsExtensible]] ( )</h1>
-        <p>When the [[IsExtensible]] internal method of a module namespace exotic object _O_ is called, the following steps are taken:</p>
+        <p>The [[IsExtensible]] internal method of a module namespace exotic object _O_ takes no arguments. It performs the following steps when called:</p>
```

Comment:
`O` is never referenced. Why name it? Same below.

Reply:
It was referenced in the original; I don't this PR needs to remove it.

Reply:
I think this PR should remove it. Why commit preambles with stuff we don't want, when we can just get them right in this PR?

Reply:
Ditto `[[PreventExtensions]]` and `[[Set]]`.